release-notes/25.03/index.html

```diff
@@ -0,0 +1,150 @@
+                </div>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <main class="flex-grow-1">
+      <section class="container py-3">
+        <div class="bg-light border rounded-3 p-4">
+          <h1>CheriBSD 25.03 Release Notes</h1>
+          <h2>Overview</h2>
+          <p>
+          Welcome to the Spring 2025 (25.03) release of the CheriBSD operating system.  As well as a general update of the baseline FreeBSD OS (corresponding to CURRENT as of January 2024) from which CheriBSD is derived, we have introduced several new research components which will be of interest to the CHERI and CheriBSD community:
+          </p>
+          <ul>
+            <li>
+            Library Compartmentalization (<a href="https://man.cheribsd.org/cgi-bin/man.cgi/release-25.03/c18n">https://man.cheribsd.org/cgi-bin/man.cgi/release-25.03/c18n</a>) has matured further and we are generally able to run with it enabled system wide.  It can be enabled system wide by setting the <code>security.cheri.lib_based_c18n_default</code> sysctl to <code>1</code>.  A new <code>procstat</code> command <code>compartments</code> allows instantiated compartments to be displayed.  While this implementation is now capable of running the full desktop, it is neither complete nor intended to be secure.
+            </li>
+            <li>
+	    Sub-library Compartmentalization has been added.  When compiled with teh <code>llvm-morello-c18n</code> compiler package and the <code>WITH_COMPARTMENTALIZTION_POLICY</code> make option set, libc will be compiled with a series of compartments.  Of particular interest, malloc and the malloc revocation wrapperm (mrs) are placed in their own compartments, protecting their powerful capabilities to the entire heap from the rest of libc.  Future releases will expand the use of such engineered compartments.
+            </li>
+            <li>
+	    To strengthen library compartmentalization, we have altered the function pointer ABI, introducing new relocations to allow function pointers to be wrapped by trampolines.  This functionality is currently disabled by default to easy source upgrades. It can be enabled system wide by setting the <code>security.cheri.lib_based_c18n_wrap_fptr</code> sysctl to <code>1</code>.  It will be the default in the next release.
+            </li>
+	    <li>
+		    System calls are now limited to the runtime loader a libsys library seperate from libc.  With compartmentalization enabled, this greatly limites the ability of arbitrary code to make system calls.
+	    </li>
+            <li>
+            User level heap temporal safety has seen a number of performace enhancements and stability improvements.
+            </li>
+	    <li>
+	    The 9pfs filesystem is now supported for CheriBSD guest OSes.
```

Comment:
Why "guest"? It's supported for any CheriBSD instance, whether in bhyve or not?

Reply:
Hmm, I guess because 9pfs over virtio is only available via hypervisors?  My original bullet didn't mention guest but did mention VirtIO (as it doesn't support 9pfs over other transports, just VirtIO).